app/ui/client/views/app/room.js

```diff
@@ -697,7 +697,7 @@ Template.room.events({
 		const hasMoreNext = RoomHistoryManager.hasMoreNext(this._id);
 
 		if ((isLoading === false && hasMore === true) || hasMoreNext === true) {
-			if (hasMore === true && lastScrollTop <= height / 3) {
+			if (hasMore === true && lastScrollTop <= height / 4) {
```

Comment:
Assign the `height / 4` expression to a variable to make it more explicit as a scrolling threshold.